doc/ref_arch/openstack/chapters/chapter05.rst

```diff
@@ -36,8 +36,8 @@ Version" refers to this maximum microversion specified for the OpenStack
 Wallaby release. Please note that in Reference Conformance (RC-1)
 testing, the System Under Test (SUT) can utilise newer microversions
 because of the OpenStack microversion policies. As per multiple
-OpenStack services documentation, for example the `Compute
-Service <https://docs.openstack.org/api-guide/compute/microversions.html>`__,
+OpenStack services documentation, for example the Compute Service
+:cite:p:`oscomputemicroversions`,
```

Comment:
would it be consistent to use abbreviations such as "ostk" for all OpenStack-elated documents, for example in this case, :cite:p:`ostk_compute_microversions`